docs/detections/api/rules/rules-api-create-rule-default-exception-list.asciidoc

```diff
@@ -0,0 +1,105 @@
+[[exceptions-api-create-rule-default-exception-list]]
+=== Create default exception list for a rule
+
+Creates a default exception list for the rule you specify. Default exception lists can only be associated with a single rule.
+
+To add exception items to a default exception list, pass in exceptions items that you want applied to the rule. Refer to <<exceptions-api-create-rule-default-exception-item>> for more information. 
+
+When an exception item’s query evaluates to `true`, the associated rule does not issue alerts even when its other criteria are met.
+
+NOTE: Default exception lists do not display on the **Shared Exception Lists page in the {security-app} UI. They only appear in the Rule exceptions on the rule details page. Refer to <<manage-exception>> to learn more.
+
+==== Request URL
+
+`POST <kibana host>:<port>/api/detection_engine/<rule_id>/exceptions`
+
+==== Request body
+
+A JSON object with these fields:
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`description` |String |Describes the exception container. |Yes
+|`list_id` |String |Unique identifier. |No, automatically created when it is not
+provided.
+|`meta` |Object |Placeholder for metadata about the list container. |No
+|`name` |String |The exception container's name. |Yes
+|`namespace_type` |String a|Determines whether the exception container is available in all {kib} spaces or just the space in which it is created, where:
+
+* `single`: Only available in the {kib} space in which it is created.
+* `agnostic`: Available in all {kib} spaces.
+
+|No, defaults to `single`.
+|`tags` |String[] |String array containing words and phrases to help categorize
+exception containers. |No
+|`type` |String a|The type of exception, which must be:
+
+* `rule_default`: Exception list that belongs to a single rule
+
+|Yes
+
+|==============================================
+
+===== Example requests
+
+Creates an exception container for holding trusted Linux process exception
+items:
+
+[source,console]
+--------------------------------------------------
+POST api/exception_lists
```

Comment:
Thanks so much for adding these docs! I'm wondering if this page is needed - we really just want users to utilize the `POST /api/detection_engine/rules/<id>/exceptions` endpoint and specify items they want added to a rule. The idea with that endpoint was to not have users need to worry about creating the "parent" or "container" part of the exception list. 

I also don't think this example matches the endpoint - /api/detection_engine/<rule_id>/exceptions.

Reply:
Hm, good question. I'm unsure whether public API endpoints _always_ need to be documented, especially when another endpoint does the same thing but in a slightly different way. Is there a way to determine whether any of our users are using this endpoint? Also, do you know if this endpoint is still being maintained and if it will be updated in the future?

Reply:
Here's the fixed path in case we keep this page:

```suggestion
POST api/detection_engine/<rule_id>/exceptions
```

Reply:
@yctercero I checked with one of our API writers, and the general advice is that public endpoints should be doc'd so we should prob keep this page.

Reply:
This might no longer be an issue since we're directing users towards the new reference site, which will/should have all the latest API docs for public Security endpoints.